### PR TITLE
Use 'react-dom/test-utils' package instead of 'react-addons-test-utils'

### LINKED ISSUE #459: Use 'react-dom/test-utils' package instead of 'react-addons-test-utils'
`react-addons-test-utils` has been deprecated and has been put in the `react-dom` package since [React v15.5.0](https://facebook.github.io/react/blog/#react-test-utils).

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Closes #459

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "build/react-motion.map"
   ],
   "dependencies": {
-    "react": "^0.14.9 || ^15.3.0"
+    "react": "^15.5.0"
```
Comment (nkbt) [inline bower.json]: Since it is only related to tests, maybe worth keeping peer deps as is, but only change devDeps?

Comment (MichaelDeBoey) [inline bower.json]: OK  no problem :-)

Comment (nkbt) [inline bower.json]: Thanks! 15.5 would be probably too harsh, I doubt that many projects got react update. Actually our projects are still on previous 15.x release.

**Comment by MichaelDeBoey** [comment]
This also fixes #456 I think?

**Comment by nkbt** [comment]
#456 requires react@16 in peer deps to be closed. Will need to check that it actually works fine with 16 with no issues.

**Comment by MichaelDeBoey** [comment]
@nkbt `React v16.0` isn't released yet, but everything that works in `v15.5.0`, [will keep working](https://facebook.github.io/react/blog/#new-deprecation-warnings) in `v16.0`

**Comment by nkbt** [comment]
alpha is there

```
23:56 $ npm info react dist-tags

{ latest: '15.5.4',
  '0.10.0-rc1': '0.10.0-rc1',
  '0.11.0-rc1': '0.11.0-rc1',
  next: '16.0.0-alpha.11',
  dev: '15.5.0-rc.2',
  '0.14-stable': '0.14.9' }
```

**Pull request merged**